src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
```

Comment:
It's a good practice to create an object with constants for your CSS classes. It will make your code more readable and maintainable. For example, you can create an object `CSS_CLASSES` and use it like `CSS_CLASSES.notification` and `CSS_CLASSES[type]`.